api-reference/docstubs/api/group-list-extensions.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "List extensions"
+description: "Get the extensions from the extensions navigation property."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List extensions
```

Comment:
Hm, @dkershaw10 , I wasn't aware that we should have documented a topic for "list open type extensions".

Reply:
Speaking of open type extensions, @wandeg , @dkershaw10 , it would be interesting to see the generated set of topics for the openTypeExtension entity (which derives from the extension abstract type).
It would be interesting because:
(1) Because extension is an abstract type, we only documented extension.md for completenesss (so that we can explain that openTypeExtension is derived from the extension base type). We did not intend to expose anything more (like GET operation) that support the extension base type. Instead, for example, we expose a GET topic for openTypeExtension. So it would be interesting to see CRUD operation topics for openTypeExtension.

(2) Because there are 9 entities that support openTypeExtension, currently on the live site, we do a 1:9 mapping - 1 topic that documents the syntax and behavior for 9 entities. I recall Dan said we can use the !INCLUDE file mechanism to actually document and maintain one source file, have the other 8 use an !INCLUDE, rather than having the same content copied and pasted in the 9 topics. So it would be interesting to see how this would look.

Reply:
@wandeg Yeah - I agree on what (2) would look like.  Not sure if you implemented that include mechanism yet James?

On the topic - Angela - we just generate based on what's in the CSDL.  We see an extensions nav property, so we create the associated docs to go with it - just like any other nav property.  We can't tell that extensions is necessarily special - and in fact it probably isn't. It's just we've treated it as something special.